test/common/testlib.py

```diff
@@ -525,7 +525,7 @@ def click_action_btn(self, sel, entry=None):
         else:
             self.click(sel + ' button:first-child')
 
-    def try_login(self, user=None, password=None, superuser=True):
+    def try_login(self, user=None, password=None, superuser=True, legacy_authorized=None):
```

Comment:
This change should be moved to the commit that removes the checkbox on the login page.